### DISCUSSION TITLE
opentrack 2022.1.0

### DISCUSSION THREAD

**Discussion by sthalik** [opentrack/opentrack#1402] [Announcements]
### Fixes

- fix a bug introduced in opentrack 2021.1.3 that caused choppy output with the PT tracker (thanks to @dglava, @Vortex76, @Len62, @animusVoxxx, @3itZ as well as the Russian IL2 forum). it should work as well as 2021.1.2 even when not using the point filter (see 4b77e13d, f51d5339, 32720488).
- grayscale conversion methods are now clearly described in the PT tracker. obsolete averaging method is gone, and hardware-based grayscaling with the ps3eye open driver is optional and off by default (see f531c316).
- fix a bug where any selection for freetrack output other than 'enable both' caused both registry keys to be cleared (see 7c8fd629).
- the neuralnet tracker works with no VC++ redistributables installed, particularly on Windows 7 (build only fix).

### Features

- module dialogs are now drawn inside the options window. for now, only few modules support it. sponsored by TrackHat Ltd.

<hr /><em>This discussion was created from the release <a href='https://github.com/opentrack/opentrack/releases/tag/opentrack-2022.1.0'>opentrack 2022.1.0</a>.</em>

**Comment by dglava** [comment]
After some quick testing, I am happy to say that this version is working fine, i.e. as good as opentrack-2.3.13. The shakiness/stuttering is gone and everything seems smooth. I even tried turning the "point filter" on and it seems to work and actually smooth out the movement a bit. Now I just have to fine tune it a bit more.

Just one small issue: the taskbar icon seems weird. Is this intended or a bug?
Edit: just noticed that the (bigger) icon says TrackHat. Is this a rebranding?

![](https://i.ibb.co/SsFVNLG/Screenshot-2022-01-07-190855.png)

**Comment by Len62** [comment]
Working great here. Did notice the weird icon myself.

**Comment by sthalik** [comment]
opentrack isn't getting rebranded or sold. I was using the same build directory for the private trackhat branch and it apparently reused the compiled icon. The build system was meant to prevent that, go figure.

The trackhat device's module will get included in opentrack once they make it official.
